docs/data/chartsApiPages.ts

```diff
@@ -1,6 +1,6 @@
 import type { MuiPage } from 'docs/src/MuiPage';
 
-const apiPages: MuiPage[] = [
+const chartsApiPages: MuiPage[] = [
```

Comment:
I've renamed the files and variables to avoid the `filenames/match-exported` ESLint error.